### ISSUE TITLE
`::Integer.to_s` method  doesn't accept Symbol

### ISSUE THREAD

**Issue by POPPIN-FUMI** [ruby/gem_rbs_collection#96]
```ruby
num = 1000.to_s(:delimited)
```

This line gets error below;

```bash
::Hash[::Symbol, ::String]

Cannot find compatible overloading of method `to_s` of type `::Integer`
Method types:
  def to_s: () -> ::String
          | (2) -> ::String
          | (3) -> ::String
          | (4) -> ::String
          | (5) -> ::String
          | (6) -> ::String
          | (7) -> ::String
          | (8) -> ::String
          | (9) -> ::String
          | (10) -> ::String
          | (11) -> ::String
          | (12) -> ::String
          | (13) -> ::String
          | (14) -> ::String
          | (15) -> ::String
          | (16) -> ::String
          | (17) -> ::String
          | (18) -> ::String
          | (19) -> ::String
          | (20) -> ::String
          | (21) -> ::String
          | (22) -> ::String
          | (23) -> ::String
          | (24) -> ::String
          | (25) -> ::String
          | (26) -> ::String
          | (27) -> ::String
          | (28) -> ::String
          | (29) -> ::String
          | (30) -> ::String
          | (31) -> ::String
          | (32) -> ::String
          | (33) -> ::String
          | (34) -> ::String
          | (35) -> ::String
          | (36) -> ::String
          | (::int) -> ::String(Ruby::UnresolvedOverloading)
```

**Comment by pocke** [comment]
The format argument is defined by ActiveSupport, and it has been deprecated since Rails 7.
https://github.com/rails/rails/blob/87d65e9b1279be17cede6be3d663a3e7ea168b75/activesupport/lib/active_support/core_ext/numeric/conversions.rb
https://github.com/rails/rails/blob/87d65e9b1279be17cede6be3d663a3e7ea168b75/activesupport/lib/active_support/core_ext/numeric/deprecated_conversions.rb


I think the overload signature should be added for active support 6.x but probably we shouldn't add it to AS 7.x.

**Comment by pocke** [comment]
This issue still exists so I'll reopen it.
ref: https://github.com/ruby/gem_rbs_collection/pull/104#issuecomment-1001899526

**Comment by pocke** [comment]
Finally this problem has been fixed by #105.
